.github/workflows/nominatim-docker-test.yml

```diff
@@ -30,5 +26,5 @@ jobs:
     - name: Workflow test
       run: echo Smoke test
 
-    - name: Initialize nominatim 
-      run: docker-compose -f docker-compose.yml up --exit-code-from web-server
+    - name: Test nominatim 
+      run: docker-compose -f ../../emission/integrationTests/docker-compose.yml up --exit-code-from web-server
```

Comment:
Not sure you need the `../..` at the beginning. This is running from the checked out directory

Reply:
Removed :)